projectile.el

```diff
@@ -3472,13 +3472,21 @@ An open project is a project with any open buffers."
                              (list (abbreviate-file-name (projectile-project-root))))
     projects))
 
+(defun projectile--move-current-project-to-end (projects)
```

Comment:
I think you should add some defcustom for selecting this behavior or the old one (as discussed in the ticket). It can be named `projectile-current-project-on-switch` with values like `'remove`, ''move-to-end`, `'keep`. 

I also think this function would benefit from an unit test.